pages.it/common/ed.md

```diff
@@ -1,7 +1,7 @@
 # ed
 
 > L'originale editor di testo Unix.
-> Maggiori informazioni: <https://man.archlinux.org/man/ed.1>.
+> Maggiori informazioni: <https://manned.org/ed>.
```

Comment:
```suggestion
> Maggiori informazioni: <https://www.gnu.org/software/ed/manual/ed_manual>.
```
I think that's the official source.

Reply:
I'm not so sure. [Wikipedia](https://en.wikipedia.org/wiki/Ed_(text_editor)) refers to it as "the version of `ed` provided by GNU".

The main link from that article seems to be [this one](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/ed.html). It seems to be as complete as the GNU manual page (I didn't check exhaustively). WDYT?

Reply:
I don't know anything about the history of `ed`, but I choose the GNU version, since I think they're doing pretty well on documentations.

However, if you think the other version is more official or prefer it for other reasons I'm fine with that.

Reply:
I don't have a strong opinion or solid knowledge. I was hoping someone could present more straightforward evidence that would help us decide :)

That said, the default page that manned.org shows is that of GNU ed anyway, so I guess it's a strict improvement to go from a man page of a tool to an actual manual of the same tool. Will change as you suggest. 👍

Reply:
[Done](https://github.com/tldr-pages/tldr/pull/5790/commits/4d1bfa817d142ae8db52f6c5690d64e909019689).

Reply:
As with `sed` I think we sould use the GNU page, since it has a whole manual about `ed` as opposed to just a man page by `manned.org`.